armbotoffboard/robotcontainer.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+
+import wpilib
+import constants
+import commands2.button
+
+from subsystems.armsubsystem import ArmSubsystem
+from subsystems.drivesubsystem import DriveSubsystem
+
+class RobotContainer:
+    def __init__(self) -> None:
+        """
+        This class is where the bulk of the robot should be declared. Since Command-based is a
```

Comment:
This docstring should belong to the RobotContainer class, not the `__init__` statement.

Reply:
sure!